carriage-user-ios/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Storyboard is not good for code review. Most teams programmatically manipulates the user interface since those code is easier to understand than XML generated by storyboard.
Not really push blocking for this diff, but we want to eventually address it in later diffs.